contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -182,3 +182,18 @@ As part of this work, the following chart inputs have changed:
 - `clickhouse.replication` was removed
 
 If you are overriding any of those values, please make the corresponding changes before upgrading. Otherwise **there's nothing you need to do**.
+
+### Upgrading from 13.x.x
+
+14.0.0 fixes customizing PostgreSQL installation, previously many `values.yaml` values did not work as expected.
+
+As part of this work, the following chart inputs have changed:
+
+- `postgresql.postgresqlHost` -> `externalPostgresql.postgresqlHost`
```

Comment:
Nit: (probably a comment for the other PR) but what about `externalPostgresql.host`? This is in line with the other external values we provide